File: swagger/models/update_client_response.py
from .base_model import BaseModel

from .client import Client

class UpdateClientResponse(BaseModel):
	"""
	no description available

	Attributes:

		client
		Contains information about the updated client.
	"""

	input_map = {
		'Client': ('client', Client),
		}

	output_map = {
		'client': 'Client',
		}

		'Client': ('client', Client),

File: swagger/Models/UpdateClientResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class UpdateClientResponse extends BaseModel
{
	/**
	 * Contains information about the updated client.
	 * 
	 * @var Client 
	 */
	public $client;

	protected function getInputMap()
	{
		return [
			'Client' => ['client', Client::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'client' => ['Client', Client::class, null],
			];
	}
}
